contracts/StakingRewardsV2.sol

```diff
@@ -81,9 +75,6 @@ contract StakingRewardsV2 is
     /// @notice summation of rewardRate divided by total staked tokens
     uint256 public rewardPerTokenStored;
 
-    /// @inheritdoc IStakingRewardsV2
-    uint256 public cooldownPeriod;
```

Comment:
are we redeploying staking v2 or upgrading? because im pretty sure if we upgrade this removal will brick everything